packages/data/src/reports/actions.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Internal dependencies
+ */
+import { getResourceName } from '../utils';
+import TYPES from './action-types';
+
+export function setItemError( endpoint, query, error ) {
```

Comment:
You're right.  I think the naming here is confusing too as it we really should separate this into queries and associated items, but might do this in a follow-up.

Reply:
Updated to `setReportItemsError` and `setReportStatsError` for the time being.  Might be worth refactoring at some point to separate the queries and actual items into separate pieces (see notes data store) but since these don't get updated right, there's probably minimal performance gain here.